docs/faq/proof-of-stake/stake-pools.md

```diff
@@ -4,11 +4,10 @@
 
 #### **1. Are there any other benefits to a stake pool other than not needing to run a full node and keeping your wallet unlocked? For example, will it have a better chance of winning or anything like that?**
 
-Well first keep in mind that wallet (`dcrwallet`) != core network (`dcrd`). In order to stake you need to have `dcrwallet` online and unlocked 24/7.
+Stake pools will usually implement multi-wallet redundancy by having many wallets physically distributed around the globe. This means there's less chance of a vote
```

Comment:
Are any of the stakepools actually globally distributed?  I haven't looked in to how they are setup, but want to confirm that this is the case before we say it.